### ISSUE TITLE
settings.json throws validation errors in latest insiders

### ISSUE THREAD

**Issue by smlombardi** [Microsoft/vscode#55151]
Issue Type: <b>Bug</b>

After las tinsiders update, status bar displays and error.  Clicking the error shows
```
{
	"resource": "/0/settings.json",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": "513",
	"severity": 8,
	"message": "Property expected",
	"source": "jsonc",
	"startLineNumber": 673,
	"startColumn": 1,
	"endLineNumber": 673,
	"endColumn": 2
}
````

<img width="535" alt="screen shot 2018-07-26 at 9 18 57 am" src="https://user-images.githubusercontent.com/835148/43264919-5ec8f63e-90b5-11e8-98b8-9a5ac5153cf0.png">

VS Code version: Code - Insiders 1.26.0-insider (e8b1ee0b4d016bc38bbc1cf06d6d1868760d5fe8, 2018-07-26T05:12:05.252Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2800)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (1.12GB free)|
|Process Argv|/Users/st016lo/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (26)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|1.2.6
Bookmarks|ale|9.0.2
project-manager|ale|8.0.0
bracket-pair-colorizer|Coe|1.0.59
vscode-eslint|dba|1.4.12
tslint|eg2|1.0.34
vscode-great-icons|emm|2.1.37
prettier-vscode|esb|1.5.0
auto-rename-tag|for|0.0.15
path-autocomplete|ion|1.12.0
svg|joc|0.1.1
Angular2|joh|6.1.1
ecdc|mit|0.12.0
HTMLHint|mka|0.5.0
theme-monokai-pro-vscode|mon|1.1.8
vscode-scss|mrm|0.6.2
vscode-stylefmt|mrm|2.5.0
angular2-inline|nat|0.0.17
incrementor|nms|0.1.0
vscode-versionlens|pfl|0.21.1
quicktype|qui|12.0.5
code-settings-sync|Sha|3.0.0
stylelint|shi|0.38.1
rewrap|stk|1.9.1
sort-lines|Tyr|1.7.0
change-case|wma|1.0.0

(4 theme extensions excluded)

</details>

**Comment by smlombardi** [comment]
This one, also:
```
{
	"resource": "/0/settings.json",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": "513",
	"severity": 8,
	"message": "Property expected",
	"source": "jsonc",
	"startLineNumber": 2746,
	"startColumn": 1,
	"endLineNumber": 2746,
	"endColumn": 2
}
```

**Comment by acrixl** [comment]
same error for build from master

**Comment by kieferrm** [comment]
Was fixed with dae69dbf00357ff8a3fe740a1f211d66b0b4681c.
